XLS-47d-PriceOracles/README.md

```diff
@@ -0,0 +1,403 @@
+  - `QuoteAsset` represents the secondary or quote asset in a trading pair. It denotes the price of one unit of the base asset. The quote asset's value is expressed in terms of the base asset. Any valid identifier such as a currency or a crypto-currency code, should be allowed and interpreted exactly like other asset identifiers in the ledger. For example, in the pair BTC/USD, USD is the quote asset; in 912810RR9/BTC, BTC is the quote asset. A new enum value STI_CURRENCY is introduced to support the `CURRENCY` field (see Appendix for details). The `BaseAsset` and `QuoteAsset` together form a trading pair, and their relationship determines the price at which one asset can be exchanged for another.
+  - `AssetPrice` is the scaled asset price, which is the price value after applying the scaling factor. This is an optional field. It is not included if the last update transaction didn't include the `BaseAsset`/`QuoteAsset` pair.
+  - `Scale` is the price's scaling factor. It represents the price's precision level. For instance, if `Scale` is `6` and the original price is `0.155` then the scaled price is `155000`. Formally, $scaledPrice = originalPrice*{10}^{scale}$. Valid `Scale` range is {0-10}. This is an optional field. It is not included if the last update transaction didn't include the `BaseAsset`/`QuoteAsset` pair.
+
+- `URI` is an optional [URI](https://datatracker.ietf.org/doc/html/rfc3986) field to reference price data off-chain. It is limited to 256 bytes.
+- `AssetClass` describes a type of the assets, for instance "currency", "commodity", "index". It is a string of up to sixteen ASCII hex encoded characters (0x20-0x7E).
+- `LastUpdateTime` is the specific point in time when the data was last updated. The `LastUpdateTime` is represented as Ripple Epoch - the number of seconds since January 1, 2000 (00:00 UTC).
+- `PreviousTxnID` is the hash of the previous transaction to modify this entry (same as on other objects with this field).
+- `PreviousTxnLgrSeq` is the ledger index of the ledger when this object was most recently updated/created (same as other objects with this field).
+
+##### The `PriceOracle` Object ID Format
+
+We compute the `PriceOracle` object ID as the SHA-512Half of the following values, concatenated in order:
+
+- The Oracle space key (0x52)
+- The Owner Account ID, `Owner`.
+- The Oracle Document ID, `OracleDocumentID`. This field describes a unique Price Oracle instance for the given account. The Oracle Document ID is maintained by the Oracle Provider.
+
+The `Owner` and `OracleDocumentID` uniquely identify the `PriceOracle` object and must be passed to the Oracle transactions.
+
+#### Example of `PriceOracle` JSON
+
+    {
```

Comment:
nit: for ease of reading, perhaps the hex-encoded fields should have comments as to what their decoded values are.